src/Tests/ViewModel/PropertyPathTests.cs

```diff
@@ -27,9 +27,9 @@ public void PropertyPath_GetPropertyPath()
 
             viewModel.Context.ModelState.ValidationTarget = viewModel;
 
-            var error = viewModel.AddModelError(vm => vm.Users[0].Post.PostText, "Testing validation error.");
+            var error = ValidationErrorFactory.AddModelError(viewModel, vm => vm.Users[0].Post.PostText, "Testing validation error.");
 
-            Assert.AreEqual("Users()[0]().Post().PostText", error.PropertyPath);
+            Assert.AreEqual("Users/0/Post/PostText", error.PropertyPath);
```

Comment:
Aren't the path supposed to start with /?

Reply:
This test does not run the `ValidationErrorPathExpander` thats why it is not absolute. This is basically testing internal API - user would not see this. It just tests extracting paths from lambda functions